### DISCUSSION TITLE
Show the older versions in dependency review

### DISCUSSION THREAD

**Discussion by jeremymeng** [community/community#118585] [Code Security]
### Select Topic Area

Product Feedback

### Body

It would be nice to know from which version the upgrade is

currently only the new version is displayed.

![image](https://github.com/community/community/assets/7583839/3e7ea10a-c596-46b3-bab1-3fee2b1c5d2c)

**Comment by uiolee** [comment]
agree

**Comment by jeremymeng** [comment]
We wouldn't know whether the update is a downgrade or upgrade without the old version.
